core/Api/exportCsvApi.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace ICT\Core\Api;
+
+/* * ***************************************************************
+ * Copyright © 2016 ICT Innovations Pakistan All Rights Reserved   *
+ * Developed By: Nasir Iqbal                                       *
+ * Website : http://www.ictinnovations.com/                        *
+ * Mail : nasir@ictinnovations.com                                 *
+ * *************************************************************** */
+
+use ICT\Core\Account;
+use ICT\Core\Api;
+use ICT\Core\CoreException;
+use ICT\Core\User;
+use ICT\Core\User\Permission;
+use SplFileInfo;
+
+class ExportCsvApi extends Api
+{
+
+
+
+  /**
+   * Export Csv
+   *
+   * @url GET /export/users
+   * 
+   */
+  public function export_list($query = array())
```

Comment:
Move this code into UserApi.php and drop this file
also rename it to export_csv, as we are going to delete the following function with the same name

Reply:
Moved